pkg/internal/api/client.go

```diff
@@ -300,27 +335,13 @@ func (c *ClientImpl) UpdateServicePassword(serviceId string, u ServicePasswordUp
 	return &serviceResponse, nil
 }
 
-func (c *ClientImpl) GetServiceStatusCode(serviceId string) (*int, error) {
-	req, err := http.NewRequest("GET", c.getServicePath(serviceId, ""), nil)
-	if err != nil {
-		return nil, err
-	}
-
-	statusCode, err := c.checkStatusCode(req)
-	if err != nil {
-		return nil, err
-	}
-
-	return statusCode, nil
-}
-
 func (c *ClientImpl) DeleteService(serviceId string) (*Service, error) {
 	service, err := c.GetService(serviceId)
 	if err != nil {
 		return nil, err
 	}
 
-	if service.State != "stopped" && service.State != "stopping" {
+	if service.State != StatusStopped && service.State != StatusStopping {
```

Comment:
Does it make sense to rename these constants from StatusXXX to StateXXX? To make sure naming is consistent with what is shown in CP?

Reply:
ok will rename everything.